kata-deploy/action/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+FROM microsoft/azure-cli:latest
+
+LABEL com.github.actions.name="Test kata-deploy in an AKS cluster"
+LABEL com.github.actions.description="Test kata-deploy in an AKS cluster"
+
+# Default to latest validated AKS-engine version
+ARG AKS_ENGINE_VER="v0.42.0"
```

Comment:
I'd still like to understand how that version is going to be maintained. How long do such versions "last"?

Reply:
I'm working on curling their release page in order to be able to pull this information automatically, and had made a PR for them to make it easy to pull "last"